index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio - Zeinab iqal</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="background">
+  <div class="page-flex">
+
+  <header>
+    <nav class="navbar">
+      <div><a class="github-account" href="https://github.com/zeinabiqal">ZEINAB IQAL</a></div>
+      <div class="nav-icon"><img src="./Images/Union.png" alt="header icon"></div>
+    </nav>
+  </header>
+
+  <section>
+    <div>
+      <div><h1 class="header">I'm Zeinab iqal Glad to see you!</h1></div> 
+      <div>
+      <p class="introduction">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p> </div>
+    </div>
+
+    <div class="contact">
+      <p>LET'S CONNECT</p>
+      <div class="contact-icons">
+        <ul class="icons-list">    
+          <li>
+            <a href="https://twitter.com/zeinab_iqal"><img src="Images/Vector1.png" alt="twitter icon"> </a>
+          </li>
+          <li>
+           <a href="https://www.linkedin.com/in/zeinab-iqal-27993059"> <img src="Images/Vector2.png" alt="linkedin icon"></a>
+          </li>
+          <li>
+            <a href="mailto:zeinabiqal@gmail.com"><img src="Images/Vector3.png" alt="M icon"></a>
+          </li>
+          <li>
+            <a href="https://github.com/ZeinabIqal"><img src="Images/Vector4.png" alt="github"></a>
```

Comment:
- [ ] I kindly suggest that you fix the URL for your image. The correct URL is 'Images/vector3.png' with a small letter `v` and not a capital letter `V`.